build.gradle

```diff
@@ -26,9 +26,9 @@ buildscript {
     }
 
     dependencies {
```

Comment:
The original benchmark used an older version of spotless (`3.15.0` vs our `5.10.1`). Is that a problem?

Reply:
It does not run, so I don't think that's a problem.